.circleci/config.yml

```diff
@@ -216,7 +216,12 @@ jobs:
   unit_tests:
     executor: test-executor
     steps:
-      - checkout
+      - run:
+          environment:
+            GIT_SSH_COMMAND: ssh -o StrictHostKeyChecking=accept-new
+          name: Checkout code
+          command: |
+            git clone --progress -b "$CIRCLE_BRANCH" "$CIRCLE_REPOSITORY_URL" .
```

Comment:
why can't we use `checkout` as it was before?

Reply:
i.e. I don't understand why undercover was looking for a git hash that doesn't exist in the repo, and how this fixes it

Reply:
I have added a comment - essentially checkout does something (somehow doesn't checkout the full repo). But there are no docs and its not configurable - they might have changed it recently, its hard to tell.